app/src/main/res/layout/schedule_message_dialog.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/scheduled_message_dialog_holder"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:paddingVertical="@dimen/activity_margin">
+
+    <com.simplemobiletools.commons.views.MyTextView
+        android:id="@+id/subtitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/activity_margin"
+        android:background="?attr/selectableItemBackground"
+        android:includeFontPadding="false"
+        android:padding="@dimen/tiny_margin"
+        android:text="@string/schedule_send_warning"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/dateIcon"
+        android:layout_width="@dimen/date_time_icon_size"
```

Comment:
lets use the social_logo dimension for this with small_margin padding, do not create new dimensions without a reason

Reply:
`small_margin` as padding makes the icon height somewhat bigger than the text views:

<img src="https://user-images.githubusercontent.com/36371707/195927037-3f51d5e7-cfff-4a9a-a9b8-315ab183166a.png" width=264 />

is that intended or should I use the `medium_margin` which looks like this:

<img src="https://user-images.githubusercontent.com/36371707/195927588-fe4875f5-b21b-4982-aa15-2e3c6e73742c.png" width=264 />

Reply:
I also added `R.dimen.small_icon_size` for the scheduled message icon in the thread adapter items, also required in the code to determine text padding dynamically.